cookbook/deployment/cluster/notifications.py

```diff
@@ -1,8 +1,118 @@
+
+Future work
+===========
+
+Work is ongoing to support a generic event egress system that can be used to publish events for tasks, workflows and
+workflow nodes. When this is complete, generic event subscribers can asynchronously process these vents for a rich
+and fully customizable experience.
+
+
+******************************
+Platform Configuration Changes
+******************************
+
+Setting Up Workflow Notifications
+=================================
+
+The ``notifications`` top-level portion of the FlyteAdmin config specifies how to handle notifications.
+
+As with schedules, the notifications handling is composed of two parts. One handles enqueuing notifications asynchronously and the second part handles processing pending notifications and actually firing off emails and alerts.
+
+This is only supported for Flyte instances running on AWS.
+
+Config
+------
+
+To publish notifications, you'll need to set up an `SNS topic <https://aws.amazon.com/sns/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc>`_.
+
+In order to process notifications, you'll need to set up an `AWS SQS <https://aws.amazon.com/sqs/>`_ queue to consume notification events. This queue must be configured as a subscription to your SNS topic you created above.
+
+In order to actually publish notifications, you'll need a `verified SES email address <https://docs.aws.amazon.com/ses/latest/DeveloperGuide/verify-addresses-and-domains.html>`_ which will be used to send notification emails and alerts using email APIs.
+
+The role you use to run FlyteAdmin must have permissions to read and write to your SNS topic and SQS queue.
+
+Let's look at the following config section and explain what each value represents: ::
+
+  notifications:
+    type: "aws"
+    region: "us-east-1"
+    publisher:
+      topicName: "arn:aws:sns:us-east-1:{{ YOUR ACCOUNT ID }}:{{ YOUR TOPIC }}"
+    processor:
+      queueName: "{{ YOUR QUEUE NAME }}"
+      accountId: "{{ YOUR ACCOUNT ID }}"
+    emailer:
+      subject: "Notice: Execution \"{{ workflow.name }}\" has {{ phase }} in \"{{ domain }}\"."
+      sender:  "flyte-notifications@company.com"
+      body: >
+        Execution \"{{ workflow.name }} [{{ name }}]\" has {{ phase }} in \"{{ domain }}\". View details at
+        <a href=\http://flyte.company.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}>
+        http://flyte.company.com/console/projects/{{ project }}/domains/{{ domain }}/executions/{{ name }}</a>. {{ error }}
+
+* **type**: Because AWS is the only cloud back-end supported for executing scheduled workflows in this case, only ``"aws"`` is a valid value. By default, the no-op executor is used.
+* **region**: This specifies which region AWS clients will use when creating SNS and SQS clients.
+* **publisher**: This handles pushing notification events to your SNS topic.
```

Comment:
Why italic and bold?

Reply:
I think `**some text**` is the rst syntax for bold. are you seeing something different?
![image](https://user-images.githubusercontent.com/2816689/119194910-c975f400-ba51-11eb-821f-71c903133551.png)

Reply:
Makes sense.